### PR TITLE
Hamburger menu

### PR DESCRIPTION
**Pull request by katrinageorge**
image
image

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.16.0",
     "react-scripts": "^5.0.1",
-    "react-virtuoso": "^4.6.0",
```
Comment (JasmineBautista) [inline client/package.json:19]: are these new downloads?

**Comment by JasmineBautista** [comment]
How did you go about rebasing this branch? Not sure if these changes might cause some issues for later PRs.

**Comment by JasmineBautista** [comment]
Also, Begin Scan isnt a page we had? Was this mentioned in a meeting that I missed?

**Comment by katrinageorge** [comment]
> Also, Begin Scan isnt a page we had? Was this mentioned in a meeting that I missed?

Added in begin scan as I felt like it makes the flow/logic a little smoother, but happy to remove it (was gonna raise it in the upcoming meeting) - defo could use a little more styling tho

File: client/package.json (hunk 1 above)
Comment (katrinageorge) [reply client/package.json:19]: yes

File: client/src/App.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ import PICResult6 from "./views/PICResult6";
 import PICResult7 from "./views/PICResult7";
 import Account from "./views/Account";
 import Checklist from "./views/Checklist";
+import AboutUs from "./views/AboutUs";
```
Comment (JasmineBautista) [inline client/src/App.js]: Are we still doing this page? Is this allocated to someone? Could the information on this page be transfer to an existing page?

File: client/src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        {/* <!-- Button to close the overlay navigation --> */}
+        <a href="javascript:void(0)" class="closebtn" onClick={handleCloseNav}>
+          &times;
+        </a>
+
+        {/* <!-- Overlay content --> */}
+        <div class="overlay-content">
+        <Link to="/">Home</Link>
+        <Link to="/AboutUs">About Us</Link>
+        <Link to="/Resources">Resources</Link>
```
Comment (JasmineBautista) [inline client/src/components/Navbar.js]: I thought we werent doing this anymore

File: client/src/css/BeginScan.css
Comment (JasmineBautista) [inline client/src/css/BeginScan.css:1]: this file is empty?

File: client/src/components/Navbar.js (hunk 3 above)
Comment (katrinageorge) [reply client/src/components/Navbar.js:35]: Do you mean the navbar itself? If so, I haven't heard about any plans for scrapping it so I'd assume not

File: client/src/css/BeginScan.css
Comment (katrinageorge) [reply client/src/css/BeginScan.css:1]: deleted

File: client/src/components/Navbar.js (hunk 3 above)
Comment (JasmineBautista) [reply client/src/components/Navbar.js]: Sorry, referring to line 34 and 35; about us and resources pages

File: client/src/App.js (hunk 2 above)
Comment (katrinageorge) [reply client/src/App.js]: Comment actioned: deleted

File: client/src/components/Navbar.js (hunk 3 above)
Comment (katrinageorge) [reply client/src/components/Navbar.js]: deleted

File: client/src/css/Common.css [hunk 4]
```diff
@@ -1,34 +1,33 @@
...
-}
-
 h1 {
   font-size: 40px;
   color: white;
   font-family: inter;
   padding-left: 75px;
 }
 
+hr {
```
Comment (JasmineBautista) [inline client/src/css/Common.css]: This is gonna cause a merge conflict I think

File: flask-server/__pycache__/register_user.cpython-311.pyc
Comment (JasmineBautista) [inline flask-server/__pycache__/register_user.cpython-311.pyc:1]: what does this have to do with the nav bar?

File: client/src/components/Navbar.js [hunk 5]
```diff
@@ -31,9 +31,10 @@ export const Navbar = () => {
         {/* <!-- Overlay content --> */}
         <div class="overlay-content">
         <Link to="/">Home</Link>
-        <Link to="/BeginScan">Begin Scan</Link>
         <Link to="/Login">Login</Link>
         <Link to="/SignUp">Sign Up</Link>
```
Comment (JasmineBautista) [inline client/src/components/Navbar.js:35]: line 34,35: future suggestion to remove this in a following PR and instead link to accounts where you can sign out and stuff

**Review by JasmineBautista — APPROVED** [review]
